public/docs/ts/latest/guide/router.jade

```diff
@@ -676,12 +684,11 @@ figure.image-display
 
 :marked
   Now that we have routes for our `Heroes` module, we'll need to register them with the *Router*.
-  We'll import the *RouterModule* like we did in the root NgModule, but there is a slight difference here.
+  We'll import the *RouterModule* like we did in the `AppModule`, but there is a slight difference here.
   In our root routing setup, we used the static **forRoot** method to register our routes and application level
-  service providers. Since we are in a submodule, we'll use **Router.forChild** method to only register additional routes. We do this
-  because the *Router* will combine all the provided routes from the submodule together and build our configuration. This allows us to continue defining
-  our feature-specific routes without modifying our main route configuration.
+  service providers. Since we are in a feature module, we'll use **Router.forChild** method to only register additional routes.
```

Comment:
I recommend calling more attention to this `forChild`. Perhaps make it a callout note